management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -241,7 +242,7 @@ public void test_stats_collection() throws Exception {
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
     assertThat(stats[0].getStatistic(Counter.class, "GetCounter").getValue(), equalTo(4L));
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
-    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, equalTo(1));
+    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, greaterThanOrEqualTo(1));
```

Comment:
It's ok ;-) we are still in master ;-) It's true for @gibsong's PR. 

But do not forget that in our test we only want to be sure stats goes through the whole voltron loop. We do not care about testing stats.

Also, do not forget that:

- computation frequency is at each 1 sec
- stat collect frequency is at each 1.5 sec

So first time we wait and get the stats from voltron (line 233) we have no samples in AllCacheGetCount, the stat has been collected, started, thus 0 sample.

1.5 sec later, line 239, we collect the stats again. So it means we are >= 3 sec since the beginning. So we have at least one other stat that has been computed. Anyway we WAIT and read next collected stats.

Line 245, we make sure that we have at least 1 value in the sample. We could have 2, if the stat scheduler had time to compute for the 3rd time. But anyway, the value will be also 4 because there has been no  other "get" operations.

So line 246, whether we take the first value or the last one, we do not care: it will be the same value anyway.